<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="exportPushButton">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QFormLayout" name="formLayout_4">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="fileNameLabel_2">
       <property name="text">
        <string>File name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="fileNameWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="fileNameLineEdit">
          <property name="text">
           <string>./gearFactory.svg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileNamePushButton">
          <property name="text">
           <string>Choose...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="formatLabel">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="formatComboBox">
       <item>
        <property name="text">
         <string>SVG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PDF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PNG</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>exportPushButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>fileNameLineEdit</tabstop>
  <tabstop>fileNamePushButton</tabstop>
  <tabstop>formatComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
